docs/code-coverage.md

```diff
@@ -1,6 +1,6 @@
 ## Code coverage
 
-We use the jacoco test coverage plugin, which will generate coverage data whenever tests are run.
+We use the jacoco test coverage plugin, which generates coverage data whenever tests are run.
 
 To run the report:
 ```
```

Comment:
I'd delete this.  It probably works but we never use it so can't be sure.

Reply:
Deleted